x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/_legend.scss

```diff
@@ -0,0 +1,38 @@
+.mlJobMapLegend__container {
+  background-color: $euiColorLightestShade;
+}
+
+.mlJobMapLegend__indexPattern {
+  height: $euiSizeM;
+  width: $euiSizeM;
+  background-color: '#FFFFFF';
```

Comment:
I suspect these styles don't take into account the dark theme

Reply:
Dark theme handling has been added to the meta issue and will done in a follow up. 👍